src/3d/p5.Renderer3D.Immediate.js

```diff
@@ -127,10 +127,9 @@ function(mode, isCurve, isBezier,isQuadratic, isContour, shapeKind){
 p5.Renderer3D.prototype._bindImmediateBuffers = function(vertices, colors){
   this._setDefaultCamera();
   var gl = this.GL;
-  var shaderKey = this._getCurShaderId();
-  var shaderProgram = this.mHash[shaderKey];
+  var shaderProgram = this._getColorVertexShader();
```

Comment:
I think this addition is left over from an old commit.  Try pulling the latest changes from webgl-geometry-refactor again.  Assigning the color vertex shader to our current program will produce a bug any time a user calls lights or materials before beginShape() and endShape().